lib/mayaUsd/commands/baseImportCommand.cpp

```diff
@@ -44,9 +44,10 @@ MayaUSDImportCommand::createSyntax()
     // UsdMayaJobImportArgs::GetDefaultDictionary.
     syntax.addFlag(kShadingModeFlag,
                    UsdMayaJobImportArgsTokens->shadingMode.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag(kShadingConversionFlag,
-                   UsdMayaJobImportArgsTokens->shadingConversion.GetText(),
+                   MSyntax::kString, MSyntax::kString);
+    syntax.makeFlagMultiUse(kShadingModeFlag);

```

Comment:
This option is now multi (array) and takes pairs of strings (shadingMode, materialConversion)